frontend/src/game/Cols.jsx

```diff
@@ -34,14 +34,14 @@ class Cols extends Component {
     }
 
     if (card.foil) {
-      className += " foil-card "; 
+      className += " foil-card ";
       // mixmix - I broke this by absorbing .foil-card into .CardBase.-foil (see CardBase.scss)
       // personally I don't care about whether pickedc cards in the col view display their foiliness
```

Comment:
// mixmix - I broke this by absorbing .foil-card into .CardBase.-foil (see CardBase.scss)
      // personally I don't care about whether pickedc cards in the col view display their foiliness

Is this comment still valid?
On dr4ft.info I can still see foil cards in column view:
![foil](https://user-images.githubusercontent.com/9874850/110792257-71618b00-8273-11eb-8d28-6d7213cccfc2.png)

Reply:
I don't know, maybe not!